hw/dv/bfm/spi_eeprom/spi_eeprom_bfm.core

```diff
@@ -0,0 +1,30 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:systems:spi_eeprom_bfm:0.1"
```

Comment:
I'm not sure that this is the right core name. "systems" is usually used for toplevels. Maybe "dv"?